src/ArticleList.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleItem from "./ArticleItem";
+import Footer from "./Footer";
+
+const ArticleList = ({ setNumArticles, pageNumber }) => {
+	const [isLoading, setIsLoading] = useState(true);
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		setIsLoading(true);
+		axios
+			.get("https://news-app-backend.onrender.com/api/articles", {
+				params: { p: pageNumber - 1 },
+			})
+			.then(({ data }) => {
+				setNumArticles(data.total_count);
+				setArticles(data.articles);
+				setIsLoading(false);
+			})
+			.catch((error) => console.log(error));
+	}, [pageNumber]);
+
+	return (
+		<div id="articleListBox">
+			{isLoading ? (
+				<div>
+					<p id="articleLoadingText">Loading Articles</p>
+					<div className="lds-ellipsis">
```

Comment:
I can see that your divs here are for your loading animation (which looks cool!!!) - I'm not massively keen on blank divs however present in the code - you could look into the `::after` pseudo selector to generate these.
Also, as your loader has some complexity, consider extracting into a component, that way you could reuse in other areas of the app!